### DISCUSSION TITLE
[已失效][自定义主页] OpenMCIM Dashboard

### DISCUSSION THREAD

**Discussion by SALTWOOD** [Meloong-Git/PCL#4906] [自定义主页]
内容：是 MCIM 模组镜像的 Dashboard，觉得可以的话请等下一周再加，觉得不行我也会在下一周处理

~~别问为啥和 OpenBMCLAPI 的那个那么像，[主控](https://github.com/SaltWood-Studio/Open93AtHome-V3)都是兼容 OpenBMCLAPI 的，面板相似很正常罢~~
**经过重构现已与 OpenBMCLAPI Dashboard 外观有较大的不同**

作者：SaltWood_233
更新地址：`https://files.mcimirror.top/PCL`
预览：
![image](https://github.com/user-attachments/assets/3f3f4b93-503e-405d-846a-02104bb123c2)

**Comment by SALTWOOD** [comment]
可能有点空了，下周会加一些别的统计信息的显示，各位感觉可以加哪些也可以提意见（

**Comment by Joker2184** [comment]
哇哦/jpg

**Comment by LTCatt** [comment]
已在 2.8.9 加入预设！

**Comment by Silverteal** [comment]
友情提示：如果使用了一些 oba-bd 的代码，您可能愿意在合适的位置附加 oba-bd 的 MIT License :)

**Reply by SALTWOOD** [reply]
实际上是没有的（）因为这种仪表盘类的我感觉很难避免长得像（
整个主页的xaml都是我写的，可能找着别的主页的看了几眼，但绝不存在照搬的情况

您要是觉得太像我周末大改一波就是

**Reply by Silverteal** [reply]
> 您要是觉得太像我周末大改一波就是

这倒无所谓，外观不是专利

**Reply by SALTWOOD** [reply]
> > 您要是觉得太像我周末大改一波就是
> 
> 这倒无所谓，外观不是专利

没逝，已经改完了

**Comment by SALTWOOD** [comment]
主页已进行更新，新版本布局改动，并且加入排行榜。

预览图如下所示：
![image](https://github.com/user-attachments/assets/3f3f4b93-503e-405d-846a-02104bb123c2)

**Comment by SALTWOOD** [comment]
由于 MCIM 源被移除，z0z0r4 在后端删除了文件分发主控，所以自定义主页预设“OpenMCIM 仪表盘”无法使用，需要移除。
cc @LTCatt
